config/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "database" : {
+        "port": "27017",
+        "uri": "localhost",
+        "collection": "PokeData"
+    },
+    "twitter" : {
+        "consumer_key": "46yqEVwOOkf7dtN5Zfqxw4Nzu",
```

Comment:
oops.. sorry i didn't know the repo is public to everyone,  should have checked, will keep this in mind and have revoked the credentials